test/e2e/serial/README.md

```diff
@@ -36,3 +36,9 @@ they found it before they run.
   node matching the given name when a random node is needed.
 - `E2E_NROP_TEST_COOLDOWN` (accepts string expressing time unit, e.g. `30s`) instructs the suite to wait for
   the specified amount of time after each spec, to give tome to the cluster to settle up.
+- `E2E_NROP_DEVICE_A` (accepts string, e.g `example.com/deviceA`) declares name of a device type that exists on
```

Comment:
not a fan of this naming, but I can't think of clearly better alternative, so let's move forward this way and improve later

Reply:
I agree with you, I'll change it to E2E_NROP_DEVICE_TYPE_1 (sounds more reasonable also when using real devices) for now until we agree on a better naming, thanks for pointing this out.